package.json

```diff
@@ -59,14 +58,11 @@
     "start-exp": "yarn workspace @fluentui/react-experiments start",
     "stats:build": "gulp stats",
     "stats:save": "gulp stats:save",
-    "sync:beachball": "beachball sync --scope \"!packages/fluentui/*\"",
```

Comment:
not used

Reply:
That one is intended for manual use if the repo gets in a bad state due to publish failing partway, so I think we should keep it.